src/createrepo_c.c

```diff
@@ -1306,9 +1308,34 @@ main(int argc, char **argv)
     // Wait until pool is finished
     g_thread_pool_free(pool, FALSE, TRUE);
 
+    GHashTableIter iter;
+    gpointer key, value;
+
+    g_hash_table_iter_init(&iter, user_data.nevra_table);
+    while (g_hash_table_iter_next(&iter, &key, &value))
+    {
+        gchar *nevra = (gchar *) key;
+        GArray *locations = (GArray *) value;
+        if (locations->len > 1) {
+            g_warning("Package '%s' has duplicate metadata entries, only one should exist", nevra);
```

Comment:
`g_error()` makes it terminate immediately, which I would be somewhat fine with, except that it prevents logging information on how to fix the problem.

Reply:
```
(.env) [dalley@thinkpad test]$ createrepo_c .
Directory walk started
Directory walk done - 3 packages
Temporary output repo path: ./.repodata/
Preparing sqlite DBs
Pool started (with 5 workers)
Warning: Package 'wget-0:1.21.3-1.fc35.x86_64' has duplicate metadata entries, only one should exist
Warning:     Sourced from location: 'bar/wget-1.21.3-1.fc35.x86_64.rpm'
Warning:     Sourced from location: 'foo/wget-1.21.3-1.fc35.x86_64.rpm'
Warning:     Sourced from location: 'wget-1.21.3-1.fc35.x86_64.rpm'

```